docs/timeseries/lombscarglemb.rst

```diff
@@ -0,0 +1,413 @@
+
+.. EXAMPLE END
+
+Periodograms from :class:`~astropy.timeseries.TimeSeries` objects
+=================================================================
+
+:class:`~astropy.timeseries.LombScargleMultiband` is able to operate on
+:class:`~astropy.timeseries.TimeSeries` objects, provided the
+:class:`~astropy.timeseries.TimeSeries` object meets a formatting requirement.
+The requirement is that the flux (or magnitudes) and errors for each band are
+provided in separate columns. If instead, your
+:class:`~astropy.timeseries.TimeSeries` object has a singular flux column with
+an associated band label column, these columns may be passed directly to
+:class:`~astropy.timeseries.LombScargleMultiband` as 1-d arrays.
+
+Example
+-------
+
+.. EXAMPLE START: Loading from a :class:`~astropy.timeseries.TimeSeries` object
+
+Consider the following generator code for a
+:class:`~astropy.timeseries.TimeSeries` object. Where timeseries data is
+populated for three photometric bands (g,r,i).
+
+>>> from astropy.timeseries import LombScargleMultiband, TimeSeries
+>>> import numpy as np
+>>> import astropy.units as u
+
+>>> rng = np.random.default_rng(1)
+>>> deltas = 240 * rng.random(180)
+>>> ts1 = TimeSeries(time_start="2011-01-01T00:00:00",
+...                  time_delta=deltas*u.minute)
+
+>>> # g band fluxes
+>>> ts1['g_flux'] = [np.nan] * 180 * u.mJy
+>>> ts1['g_err'] = [np.nan] * 180 * u.mJy
+>>> y_g = 3 + 2 * np.sin(10 * np.pi * ts1['time'].mjd[0:60])
+>>> dy_g = 0.01 * (0.5 + rng.random(60)) # uncertainties
+>>> ts1['g_flux'].value[0:60] = y_g
+>>> ts1['g_err'].value[0:60] = dy_g
+>>> # r band fluxes
+>>> ts1['r_flux'] = [np.nan] * 180 * u.mJy
+>>> ts1['r_err'] = [np.nan] * 180 * u.mJy
+>>> y_r = 3 + 2 * np.sin(10 * np.pi * ts1['time'].mjd[60:120])
+>>> dy_r = 0.01 * (0.5 + rng.random(60)) # uncertainties
+>>> y_r += dy_r * rng.standard_normal(60)
+>>> ts1['r_flux'].value[60:120] = y_r
+>>> ts1['r_err'].value[60:120] = dy_r
+>>> # i band fluxes
+>>> ts1['i_flux'] = [np.nan] * 180 * u.mJy
+>>> ts1['i_err'] = [np.nan] * 180 * u.mJy
+>>> y_i = 3 + 2 * np.sin(10 * np.pi * ts1['time'].mjd[120:])
+>>> dy_i = 0.01 * (0.5 + rng.random(60)) # uncertainties
+>>> y_i += dy_i * rng.standard_normal(60)
+>>> ts1['i_flux'].value[120:] = y_i
+>>> ts1['i_err'].value[120:] = dy_i
+
+Our timeseries data is set up to be asynchronous, where a given timestamp
+corresponds to a measurement in a single band. However, if your data instead
+has one timestamp per multiple band measurements, or a mixture,
+:class:`~astropy.timeseries.LombScargleMultiband` will still be able to
+operate on it.
+
+To operate on the example :class:`~astropy.timeseries.TimeSeries`,
+:class:`~astropy.timeseries.LombScargleMultiband` has a loader function, as
+follows:
+
+>>> ls = LombScargleMultiband.from_timeseries(ts1, signal_column_name=['g_flux', 'r_flux', 'i_flux'],
+...                                           uncertainty=['g_err', 'r_err', 'i_err'],
```

Comment:
This example makes me think  this be 'uncertainty_column_name' for consistency? Or maybe it should be signal_column and uncertainty_column to make it a little more concise.